### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by eamonnotoole**

### CODE REVIEW

File: docs/resources/vmaas_instance.md [hunk 1]
```diff
@@ -126,72 +132,12 @@ resource "hpegl_vmaas_instance" "tf_instance" {
   # On creating only poweron operation is supported. Upon updation all other
   # lifecycle operations are permitted.
   power = "poweron"
+  # Restarts the instance if set to any positive integer.
+  # Restart works only on pre-created instance.`,
```
Comment (zachovich) [inline docs/resources/vmaas_instance.md:136]: Does that means this attribute is ineffective on new instances?

File: docs/resources/vmaas_instance_clone [hunk 2]
```diff
@@ -0,0 +1,206 @@
+---
```
Comment (zachovich) [inline docs/resources/vmaas_instance_clone:1]: What is that file?

Comment (eamonnotoole) [reply docs/resources/vmaas_instance_clone:1]: That needs to be removed.

File: docs/resources/vmaas_instance.md (hunk 1 above)
Comment (eamonnotoole) [reply docs/resources/vmaas_instance.md:136]: That would need to be answered by the VMaaS team.

**Review by zachovich — APPROVED** [review]

**Pull request merged**